lib/active_merchant/billing/gateways/quickpay/quickpay_v10.rb

```diff
@@ -214,7 +214,9 @@ def invalid_operation_code(response)
         end
 
         def invalid_operation_message(response)
-          response['operations'].last['qp_status_msg']
+          if response['operations']
```

Comment:
Hmm.. no, then it would return a boolean. It should return the error message from the last operation (if present)